docs/v4/objects/routing.md

```diff
@@ -154,6 +154,31 @@ $app->get('/hello/{name}', function ($request, $response, $args) {
 });
 ```
 
+<div class="alert alert-info">
+    <div><strong>Heads Up!</strong></div>
+    <p>Slim does not support <em>static closures</em>.</p>
+</div>
+
+However it is possible to create controller classes with static actions (methods).
```

Comment:
Why recommend using classes with static methods over `__invoke` ?

Reply:
Hm, I was stuck in the `static` world.

Actually we should probably write that it is possible to create controller classes without a constructor if someone wants to avoid that the container would be injected into the controller.

Reply:
Removed this section in https://github.com/slimphp/Slim-Website/pull/390/commits/f088466c6e673c3f180c7c3b47f370f6ddf74e13